specification/communication/data-plane/TrunkStatus/preview/2022-07-01-preview1/communicationservicestrunkstatus.json

```diff
@@ -0,0 +1,212 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "TrunkStatus Service",
+    "description": "TrunkStatus Service",
+    "contact": {
+      "email": "acsdevexdisc@microsoft.com"
+    },
+    "version": "2022-07-01-preview1"
+  },
+  "paths": {
+    "/sip/trunks/status": {
+      "get": {
+        "tags": [
+          "TrunksStatus"
+        ],
+        "summary": "Gets list of all the trunks SBC status for resource",
+        "operationId": "GetTrunksStatus",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
```

Comment:
The paging was not fixed properly, cannot generate code from this specification. Consider adding:
"itemName": "values"